conformance/conformance.go

```diff
@@ -77,10 +77,18 @@ const (
 // Future profiles will cover EPP and ModelServer layers.
 const GatewayLayerProfileName confsuite.ConformanceProfileName = "Gateway"
 
+// TODO() create a dedicated share location for feature names similar to
```

Comment:
nit: if there is no assignee/link for the TODO, I would just use a colon instead of empty parens

Reply:
Here and below

Reply:
Done.